libraries/azure_backend.rb

```diff
@@ -194,20 +197,24 @@ def initialize(opts)
       opts[option_name] = ENV[env_var_name] unless ENV[env_var_name].nil?
```

Comment:
It was done in this way so that thing scould easily be tested without having to modify a file.
I have worked on some CIS controls for Azure, which @jamesc asked for, that can use this concept to quickly test things.